### DISCUSSION TITLE
table column width in output console?

### DISCUSSION THREAD

**Discussion by xzhai12** [jshinonome/vscode-q#86] [Q&A]
Hey guys, recently I found that somehow my settings was changed and I couldn't have my VSCode display all content in a column whose value is a super long string....

For example, before it's like this
```
c                                                                     
-------------------------------------------------------------------------------
"a=vdavlku=frewlajkfdu=dvsa,riqwufds=df"
```

now it's like 
```
c                                                                     
-------------------------------------------------------------------------------
"a=vdavlku=frewlajkfdu=dvsa...
```

which basically ruins the coloring of columns after it due to the missing quote mark. 

Any ideas?

**Comment by xzhai12** [comment]
forgot to say that this is a great plug-in and it really levels up my work with `q`.

**Comment by jshinonome** [comment]
Hi, thanks for using vscode-q. If you want to show more chars, you have to set \c to show a wide console size. As for highlight color, I may be able to fix it. Feel free to raise a ticket.

**Reply by xzhai12** [reply]
wow thanks very much for such a prompt reply. 
I have set \c to 36 1800 already. The problem I'm having is not a limited width of q table display, but the max width of each column. I guess this "problem" is a combination of plug-in and vscode (mostly vscode I guess).
fixing the color would be awesome too. I feel like that the coloring has a search and pairing of "", which might be better if it's based on column.

**Reply by jshinonome** [reply]
Yeah, plugin also has a console size configuration. I didn't want to mess up with system console size configuration.

This is the configuration you need to change for plugin.

q-client.output.consoleSize
